src/server.rs

```diff
@@ -22,13 +25,19 @@ pub async fn run(ipc: IPC<Event>, mut shutdown_signal: Receiver<()>) {
 }
 
 async fn try_run(
-    ipc: IPC<Event>,
+    ipc_event: IPC<Event>,
     shutdown_signal: &mut Receiver<()>,
 ) -> Result<(), transport::Error> {
     log::info!("Staring gRPC Server ...");
 
     let addr = "0.0.0.0:50051".parse().unwrap();
-    RPC::builder(ipc)
+    Server::builder()
+        .add_service(MissionServer::new(RPC {
+            ipc: ipc_event.clone(),
+        }))
+        .add_service(StreamerServer::new(RPC {
+            ipc: ipc_event.clone(),
+        }))
```

Comment:
We can re-use the same RPC here

```suggestion
    let rpc = RPC::new(ipc);
    Server::builder()
        .add_service(MissionServer::new(rpc.clone()))
        .add_service(StreamerServer::new(rpc))
```

Reply:
Done